server/src/controllers/__tests__/map.spec.ts

```diff
@@ -0,0 +1,79 @@
+import MapController from '../map';
+import database from '../../db/index';
+import { Repository, Model } from 'sequelize-typescript';
+import { Map } from '../../models/map';
+import { Sandtrap } from '../../models/sandtrap';
+import { Water } from '../../models/water';
+import { Barrier } from '../../models/barrier';
+import { FullMap } from '../../types';
```

Comment:
Can you run the linter on this file, please? It should have caught this in the linter action; I will need to update that.